neo/rawio/blackrockrawio.py

```diff
@@ -930,28 +937,32 @@ def __read_nsx_header_variant_b(self, nsx_nb):
             # filter settings used to create nsx from source signal
             ("hi_freq_corner", "uint32"),
             ("hi_freq_order", "uint32"),
-            ("hi_freq_type", "uint16"),  # 0=None, 1=Butterworth
+            ("hi_freq_type", "uint16"),  # 0=None, 1=Butterworth, 2=Chebyshev
             ("lo_freq_corner", "uint32"),
             ("lo_freq_order", "uint32"),
             ("lo_freq_type", "uint16"),
-        ]  # 0=None, 1=Butterworth
+        ]  # 0=None, 1=Butterworth, -2-Chebyshev
```

Comment:
```suggestion
        ]  # 0=None, 1=Butterworth, 2=Chebyshev
```
? or is it negative 2

Reply:
No, that was a typo.